Documentation/Installation/Install.rst

```diff
@@ -126,122 +133,132 @@ Interactive / guided setup (questions/answers):
+In the event that certain criteria are not met, TYPO3 will display a list of
+issues it has detected accompanied by a resolution for each issue.
 
-Once changes have been made, TYPO3 can re-scan the host environment by reloading the page `https://example-project-site.local/typo3/install.php`.
+Once changes have been made, TYPO3 can re-scan the host environment by reloading
+the page :samp:`https://example-project-site.local/typo3/install.php`.
 
-.. include:: ../Images/AutomaticScreenshots/QuickInstall/Step1SystemEnvironment.rst.txt
+..  include:: ../Images/AutomaticScreenshots/QuickInstall/Step1SystemEnvironment.rst.txt
 
-Select a Database
+Select a database
 ~~~~~~~~~~~~~~~~~
 
 Select a database connection driver and enter the credentials for the database.
 
-.. include:: ../Images/AutomaticScreenshots/QuickInstall/Step2DatabaseConnection.rst.txt
+..  include:: ../Images/AutomaticScreenshots/QuickInstall/Step2DatabaseConnection.rst.txt
 
 TYPO3 can either connect to an existing empty database or create a new one.
 
-The list of databases available is dependent on which database drivers are installed on the host.
+The list of databases available is dependent on which database drivers are
+installed on the host.
 
-For example, if an instance of TYPO3 is intended to be used with a MySQL database then the PHP extension 'pdo_mysql' is required.
-Once it's installed, :guilabel:`MySQL Database` will be available as an option.
+For example, if an instance of TYPO3 is intended to be used with a MySQL/MariaDB
+database then the PHP extension "pdo_mysql" is required. Once it is installed,
+:guilabel:`MySQL Database` will be available as an option.
 
-.. include:: ../Images/AutomaticScreenshots/QuickInstall/Step3ChooseDb.rst.txt
+..  include:: ../Images/AutomaticScreenshots/QuickInstall/Step3ChooseDb.rst.txt
 
-Create Administrative User & Set Site Name
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create administrative user & set site name
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 An administrator account needs to be created to gain access to TYPO3's backend.
 
 An email address for this user can also be specified and a name can be given.
 
-.. include:: ../Images/AutomaticScreenshots/QuickInstall/Step4AdminUserSitename.rst.txt
+..  include:: ../Images/AutomaticScreenshots/QuickInstall/Step4AdminUserSitename.rst.txt
 
 ..  note::
     The password must comply with the configured
     :ref:`password policy <t3coreapi:password-policies>`.
 
-Initialise
------------
+Initialize
+----------
 
-TYPO3 offers two options for initialisation: creating an empty starting page or
-it can go directly to the backend administrative interface.
-Beginners should select the first option and allow TYPO3 to create an empty starting page.
+TYPO3 offers two options for initialization: creating an empty starting page or
+it can go directly to the backend administrative interface. Beginners should
```

Comment:
Not sure if this is always the case. If you plan to start with the introduction package for example it is better not to choose an empty starting page?

Reply:
Don't know. haven't used the introduction package yet. This PR is just a "I make it a little better" not a complete overhaul with checking everything written and testing all the variants. This should be done at some point in time, but currently not having time for that.

Reply:
Maybe add a todo or issue to check that later?

Reply:
As far as I remember, you have only these two options: Create an empty start page or create nothing. I assume, the introduction package is then an extension that has to be added later.